extensions/wikia/StaffSig/StaffSig.php

```diff
@@ -21,20 +21,25 @@ function wfSigSetup(&$parser) {
 
 function wfMakeStaffSignature( $contents, $attributes, $parser ) {
 	$title = GlobalTitle::newFromText('Staff', 4 /*project*/, 177);
-	return wfMakeSignatureCommon( $title->getFullURL(), "This user is a member of Wikia Staff" );
+	return wfMakeSignatureCommon( $title->getFullURL(), "This user is a member of Fandom Staff" );
 }
 
 function wfMakeHelperSignature( $contents, $attributes, $parser ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$contents, $attributes, $parser". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)